config/jenkins-jobs_infra.ci.jenkins.io.yaml

```
@@ -499,3 +499,8 @@ jobsDefinition:
             description: "docs.jenkins.io File Share Service Principal Writer"
             subscriptionId: "${JENKINSINFRA_AZURE_PRIMARY_SUBSCRIPTION_ID}"
             tenant: "${JENKINSINFRA_AZURE_TENANT_ID}"
+      stats.jenkins.io:
+        name: stats.jenkins.io
+        allowUntrustedChanges: true
```

```suggestion
        allowUntrustedChanges: false
```

or

```suggestion
```

as we discussed earlier today: we don't want a non maintainer/write to have their build run